server/server.js

```diff
@@ -2,12 +2,28 @@ var path = require('path')
 var express = require('express')
 var bodyParser = require('body-parser')
 
+var routes = require('./routes')
+
 var PORT = process.env.PORT || 3000
 var app = express()
 
 app.use(bodyParser.json())
 app.use(express.static(path.join(__dirname, '../public')))
 
+app.get('/v1/donors', routes.getDonors)
+app.get('/v1/recipients', routes.getRecipients)
+app.get('/v1/donors/:id', routes.getDonor)
+app.get('/v1/recipients/:id', routes.getRecipient)
+app.get('/v1/tickets', routes.getTickets)
+app.get('/v1/tickets/donors/:id', routes.getDonorTicket)
+app.get('/v1/tickets/recipients/:id', routes.getRecipientTicket)
+app.get('/v1/tickets/comments/:id', routes.getTicketComments)
```

Comment:
These should be:
```
/v1/tickets/:id/donors
/v1/tickets/:id/recipients
/v1/tickets/:id/comments
```
The id is attached to the ticket, not to the donor, recipient or comment. In addition, only the comments route makes sense. The tickets route should return tickets and if necessary support a query string search with something like:
```
/v1/tickets?type=donor
```